src/Hedgehog/Hedgehog.fsproj

```diff
@@ -1,45 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{B338B8C4-1C98-4F42-9C45-56BD7EF2AD0A}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>Hedgehog</RootNamespace>
-    <AssemblyName>Hedgehog</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <AssemblyVersion>0.6.0.0</AssemblyVersion>
+    <FileVersion>0.6.0.0</FileVersion>
```

Comment:
I think here we need to also take into account the generated NuGet package which right now gets generated from the [`.paket.template`](https://github.com/hedgehogqa/fsharp-hedgehog/blob/3f7f34c5e6d77e5d333ec7b0a0f8aaaf78aca573/src/Hedgehog/paket.template).

We probably need to get rid of that file and move all its content here, as attributes.

Reply:
I tried that but the package that is generated doesn't have the right versions for the dependencies (since only paket knows about the dependencies in `paket.dependencies`).

Reply:
e.g `FSharp.Core` is 4.3.